### PR TITLE
fix(ui5-checkbox): Add ui5-checkbox components default values

### PR DESCRIPTION
**Pull request by gustavokath**
- Add default value as false for checked, disabled, read-only, wrap

Fixes: #407

### CODE REVIEW

File: packages/main/test/specs/CheckBox.spec.js [hunk 1]
```diff
@@ -36,4 +36,10 @@ describe("CheckBox general interaction", () => {
 		assert.strictEqual(truncatingCbHeight, CHECKBOX_DEFAULT_HEIGHT, "The size of the checkbox is : " + truncatingCbHeight);
 		assert.ok(wrappingCbHeight > CHECKBOX_DEFAULT_HEIGHT, "The size of the checkbox is more than: " + CHECKBOX_DEFAULT_HEIGHT);
 	});
-});
\ No newline at end of file
+
+	it("tests checked default value is false", () => {
+		const checkBox = browser.findElementDeep("#cb1");
+
+		assert.strictEqual(checkBox.checked, false, "Check if default value for checked is false");
```
Comment (ilhan007) [inline packages/main/test/specs/CheckBox.spec.js]: Try with checkBox.getProperty("checked"). 
The checkbox is "wdio" (the test framework in use) wrapper for the elements in the html page .

**Comment by ilhan007** [comment]
Hello @gustavokath ,
thanks for reporting.

Yes, the boolean properties should return false by default, not undefined.

Actually, we are working on other change, not related to this issue, but addresses the issue with the boolean (and string) default values.
Now, as you created an issue and pull request, I decided to make a separate pull request, so the boolean properties default should not be an issue (https://github.com/SAP/ui5-webcomponents/pull/409) any more. 

With this pull request, all components` boolean properties should return false by default
and the developers would not have to declare defaultValue: false in the metadata, although there is such option (and you fixed the the ui5-checkbox boolean props this way). 

Furthermore, all boolean properties of the ui5 web components should be "false" by default, the can`t be "true" by default, so declaring defaultValue: false becomes redundant.

However, 
I will accept the pull request, as it also updates the JSDoc, but once the tests are green. 
About the failing test, look at the inline comment.

best,
ilhan

File: packages/main/test/specs/CheckBox.spec.js (hunk 1 above)
Comment (gustavokath) [reply packages/main/test/specs/CheckBox.spec.js]: Thanks! 😄

**Comment by gustavokath** [comment]
Hi @ilhan007,
Thanks and the build is passing now, thanks for the suggestion.

Also, this change you've done in your PR will help us that are using the ui5-webcomponents. I agree with you that having a place where all boolean values are set as default to false is a much better approach than go over all components and update them.

Thanks

**Pull request merged**